[2024-09-30T19:50:02.491+0000] {processor.py:156} INFO - Started process (PID=32) to work on /opt/airflow/dags/lnd_oppai.py
[2024-09-30T19:50:02.494+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/lnd_oppai.py for tasks to queue
[2024-09-30T19:50:02.495+0000] {logging_mixin.py:149} INFO - [2024-09-30T19:50:02.495+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/lnd_oppai.py
[2024-09-30T19:50:02.500+0000] {logging_mixin.py:149} INFO - [2024-09-30T19:50:02.499+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/lnd_oppai.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/lnd_oppai.py", line 2, in <module>
    from Senior_Data_Analyst_Challenge.ingestion_incremental.ingestion_inicial import main
ModuleNotFoundError: No module named 'Senior_Data_Analyst_Challenge'
[2024-09-30T19:50:02.501+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/lnd_oppai.py
[2024-09-30T19:50:02.640+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/lnd_oppai.py took 0.155 seconds
[2024-09-30T19:50:32.852+0000] {processor.py:156} INFO - Started process (PID=94) to work on /opt/airflow/dags/lnd_oppai.py
[2024-09-30T19:50:32.853+0000] {processor.py:799} INFO - Processing file /opt/airflow/dags/lnd_oppai.py for tasks to queue
[2024-09-30T19:50:32.853+0000] {logging_mixin.py:149} INFO - [2024-09-30T19:50:32.853+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/lnd_oppai.py
[2024-09-30T19:50:32.856+0000] {logging_mixin.py:149} INFO - [2024-09-30T19:50:32.855+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/lnd_oppai.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/lnd_oppai.py", line 2, in <module>
    from Senior_Data_Analyst_Challenge.ingestion_incremental.ingestion_inicial import main
ModuleNotFoundError: No module named 'Senior_Data_Analyst_Challenge'
[2024-09-30T19:50:32.856+0000] {processor.py:811} WARNING - No viable dags retrieved from /opt/airflow/dags/lnd_oppai.py
[2024-09-30T19:50:32.902+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/lnd_oppai.py took 0.053 seconds
